src/routes/admin/ieee8021x/ieee8021xValidator.ts

```diff
@@ -0,0 +1,58 @@
+/*********************************************************************
+ * Copyright (c) Intel Corporation 2022
+ * SPDX-License-Identifier: Apache-2.0
+ **********************************************************************/
+
+import { check, type CustomValidator } from 'express-validator'
+
+export const ieee8021xValidator = (): any => [
+  check('profileName')
+    .not()
+    .isEmpty()
+    .withMessage('802.1x profile name is required')
+    .matches('^[a-zA-Z0-9]+$')
+    .withMessage('802.1x profile name should be alphanumeric')
+    .isLength({ max: 32 })
+    .withMessage('802.1x profile name maximum length is 32'),
+  check('authenticationProtocol')
+    .not()
+    .isEmpty()
+    .withMessage('Authentication protocol is required')
+    .isIn([0, 3, 5, 10])
+    .withMessage('Authentication protocol should be either 0(EAP_TLS), 3(EAP_GTC), 5(EAPFAST_GTC) or 10(EAPFAST_TLS)'),
```

Comment:
these names/labels don't match format with those provided for the UI. I have a branch with fixes if you want to merge it in.